### ISSUE TITLE
please update glide, picasso and fresco libs

### ISSUE THREAD

**Issue by mojtaba-yekta** [VEINHORN/ScrollGalleryView#86]
Hi,
Please update image loader libs to latest versions:

```
//fresco version in fresco-loader:
implementation 'com.facebook.fresco:fresco:1.11.0'
//latest version is:
implementation 'com.facebook.fresco:fresco:2.0.0'

//glide version in glide-loader:
implementation 'com.github.bumptech.glide:glide:4.8.0'
//latest version is:
implementation 'com.github.bumptech.glide:glide:4.9.0'
annotationProcessor 'com.github.bumptech.glide:compiler:4.9.0'

//glide version in picasso-loader:
implementation 'com.squareup.picasso:picasso:2.5.2'
//latest version is:
implementation 'com.squareup.picasso:picasso:2.71828'
```

**Comment by Oziomajnr** [comment]
@mojtaba-yekta You update the library and submit a pull request.

**Comment by mirjalal** [comment]
Related #94

**Comment by VEINHORN** [comment]
Fixed by @mirjalal in #94
